pkg/apis/node/v1alpha1/conversion.go

```diff
@@ -33,11 +33,23 @@ func addConversionFuncs(s *runtime.Scheme) error {
 func Convert_v1alpha1_RuntimeClass_To_node_RuntimeClass(in *v1alpha1.RuntimeClass, out *node.RuntimeClass, s conversion.Scope) error {
 	out.ObjectMeta = in.ObjectMeta
 	out.Handler = in.Spec.RuntimeHandler
+	if in.Spec.Overhead != nil {
+		out.Overhead = &node.Overhead{}
```

Comment:
Why do we need it to be non-nil when converted?

Reply:
Just the way the auto-generated conversion function is written, it needs a non-nil object to update. Note that we only do this if the input overhead is non-nil.